tfs/src/main/java/hudson/plugins/tfs/TeamPullRequestMergedDetailsAction.java

```diff
@@ -24,7 +24,7 @@
     private static final long serialVersionUID = 1L;
     private static final String URL_NAME = "team-pullRequestMergedDetails";
 
-    public GitPullRequestEx gitPullRequest;
+    public transient GitPullRequestEx gitPullRequest;

```

Comment:
"Serializable" is not required for persisting the data on the disk. My concern is that my making the class formally serializable you may break the data persistency